com.chartboost.mediation/Runtime/Plugins/iOS/ChartboostMediationUnityBridge.mm

```diff
@@ -80,6 +80,11 @@ TObj objFromJsonString(const char* jsonString) {
     return getCStringOrNull(json);
 }
 
+const char* dictionaryToJSON(NSDictionary *data)
```

Comment:
Is it `char*` or `char *`?

Reply:
That's actually the old code that git is showing in green for some reason. I guess we use to not have the whitespace earlier(`char*`) and now we have started including it( `char *`) or it may be just a typo. I've seen both in multiple places.

@SCastanedaMunoz do we have any preference over what should be the right syntax ?